cylc/uiserver/resolvers.py

```diff
@@ -273,25 +277,51 @@ async def scan(
         cls,
         args: dict,
         workflows_mgr: 'WorkflowsManager',
-    ):
+    ) -> tuple[bool, str]:
         await workflows_mgr.scan()
         return cls._return("Scan requested")
 
     @classmethod
-    async def play(
+    async def run_command(
         cls,
+        command: Iterable[str],
         workflows: Iterable[Tokens],
         args: Dict[str, Any],
-        workflows_mgr: 'WorkflowsManager',
         log: 'Logger',
-    ) -> List[Union[bool, str]]:
```

Comment:
Changed from `list[bool | str]` to `tuple[bool, str]` as this is safer for the way we are trying to use it (`('msg', False)` would have been acceptable before).